docs/feature-flags/creating-feature-flags/trigger.mdx

```diff
@@ -0,0 +1,114 @@
+
+:::
+
+The Trigger URL includes the authentication token, so all you need to do to activate the trigger is to perform a POST request using the Trigger URL. You must keep the Trigger URL safe and secure, as it contains sensitive information. Anyone with access to the Trigger URL can modify the state of your system flag.
+
+:::warning Trigger URL linkage
+
+In the event of a Trigger URL leak, the Bucketeer team strongly recommends that you immediately reset or delete the trigger. Otherwise, anyone with the Trigger URL can modify the state of your application.
+
+:::
+
+### Usage recomendations
+
+When using Bucketeer, it's important to note that triggers don't receive any information from the Webhook request body. In simpler terms, all you need to do to execute a trigger is perform a POST request to the Trigger URL you received when creating the trigger. As a result, Bucketeer is unable to identify the source of the POST request. Therefore, if you're using different tools to manage the flag remotely, it's recommended that you create a distinct trigger for each tool. Doing this lets you easily identify the source of a flag turn-off.
+
+:::note Future releases
+
+Bucketeer has in its roadmap the objective to accept payloads in the Webhook body to enable you to provide additional information regarding the request source. Stay tuned for future releases.
+
+:::
+
+## Manage triggers
+
+When you access the Trigger tab, you will have access to all existing triggers related to the current flag. The image provides an example of a flag associated with three triggers.
+
+<CenteredImg
+  imgURL="img/feature-flags/trigger/flag-trigger-existing-triggers.png"
+/>
+
+Each trigger presents the following information:
+
+- **Action**: Specify if the trigger turns the flag ON or OFF when activated.
+- **Trigger URL**: Part of the Trigger URL. As mentioned before, you only have access to the complete trigger URL at the time of trigger creation.
+- **Triggered Times**: Inform the number of times the trigger was used and the associated action was performed. 
+- **Last Triggered**: Inform the date and time the trigger was last used.
+
+:::info Triggered execution
+
+The Triggered Times and Last Triggered will only be updated if the trigger action changes the flag state. For instance, if the flag was initially ON, and the trigger was executed to turn it OFF, then the Triggered Times and Last Triggered will be updated. However, suppose the flag was initially OFF, and the trigger action was also intended to turn it OFF. In that case, there will be no changes to the Triggered Times and Last Triggered because the flag state was not changed.
```

Comment:
The behavior has changed. When the flag is already enabled, and it tries to enable it, and vice-versa, it will just update the trigger data, including its count. So I think we can remove this information. Only when the trigger is disabled, if the URL is called, it won't do anything.